.github/workflows/release-pr.yml

```diff
@@ -99,10 +99,10 @@ jobs:
 
       - name: "Create a PR to trunk"
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.BOTWOO_TOKEN }}
           BRANCH_NAME: ${{ steps.create_branch.outputs.branch-name }}
           CHANGELOG: ${{ steps.generate_changelog.outputs.changelog }}
           RELEASE_VERSION: ${{ steps.create_branch.outputs.trimmed-version }}
         run: |
-          PR_BODY=$(echo -e ":warning: Please do not merge the PR from the GitHub interface. :warning:\n\n Instead, you can use the following command: \` git merge --no-ff $BRANCH_NAME -m 'Merge $BRANCH_NAME into trunk' \`\n\`\`\`\n${CHANGELOG}\n\`\`\`")
+          PR_BODY=$(echo -e ":warning: Please do not merge the PR from the GitHub interface. :warning:\n\n Instead, you can use the following command:\n\`\`\`\n git checkout trunk && git pull \n git merge --no-ff $BRANCH_NAME -m 'Merge $BRANCH_NAME into trunk' \n git push origin trunk \n\`\`\` \n Changelog: \n\`\`\`\n${CHANGELOG}\n\`\`\`")
```

Comment:
Do we need to create and push tags as well for the new release branch or will this suffice? 🤔

Reply:
When the PR is reviewed, and it is time to merge into trunk, the product is not released yet, so it could be too soon to create a tag and GitHub release. You could have a last-minute fix before we execute the release command somehow. 

Note that I originally wrote this workflow to also tag and create a GitHub release, but then I looked back at our process and saw that this was done during the `woorelease release` command. So I decided to play it safe and postpone this step until after the deployment is successful, in [the post-release steps workflow](https://github.com/Automattic/woocommerce-payments/blob/c6154cddf73f49e486ee9316da1075c9cf982418/.github/workflows/post-release-updates.yml#L31-L73).

Reply:
ack, makes sense.